src/components/App/index.js

```diff
@@ -43,40 +46,53 @@ const App = () => {
     setExportHref(getExportHrefValue(playList))
   }, [playList])
 
+  useEffect(() => {
+    const root = window.document.documentElement
+    root.classList.remove(colorTheme === 'dark' ? 'light' : 'dark')
+    root.classList.add(colorTheme)
+    localStorage.setItem('theme', colorTheme)
+  }, [colorTheme])
+
   return html`
-    <div class="p-4">
+    <div class="p-4 dark:bg-[#18181b] h-screen transition duration-500">
       <div class="flex align-center mb-4">
         <img class="w-10 sm:w-20 mr-3 object-contain" src=${svgLogo} alt="logo" />
-        <h1 class="font-medium leading-tight text-2xl sm:text-4xl lg:text-5xl leading-none">Anonymous youtube playlist generator</h1>
+        <h1 class="font-medium leading-tight text-2xl sm:text-4xl lg:text-5xl leading-none dark:text-slate-300">Anonymous youtube playlist generator</h1>
       </div>
-      <p class="mb-4">Create youtube playlists without a google account</p>
+      <p class="mb-4 dark:text-slate-300">Create youtube playlists without a google account</p>
       <hr class="mb-4"/>
+
+      <div class="fixed bottom-0 right-0 p-5">
+        <span onClick=${() => setColorTheme(oldColorTheme => oldColorTheme === 'dark' ? 'light' : 'dark')} class="w-10 h-10 bg-primary-color block rounded-full shadow-lg cursor-pointer text-white flex items-center justify-center transition duration-500">
+          <${colorTheme === 'light' ? light : dark} customClass="h-6 pr-[0.2rem]" width="full" fill="white" />
+        </span>
+      </div>
       
       <div class="flex items-end flex-wrap mb-4 max-w-xs sm:max-w-xl">
         <a 
-          class="${css.button} mr-2"
+          class="${colorTheme === 'light' ? css.button : css.outlineButton} mr-2"
```

Comment:
I noticed that you repeated this expression in the lines 81, 88 and 121. Can you make a reusable function that return that expression?